homework_1/task3/task3.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"github.com/pkg/errors"
+	"github.com/sethgrid/pester"
+)
+
+type pokemonAPIResponse struct {
+	Name                     string
+	Location_area_encounters string
+}
+
+type pokemonEncounters struct {
+	Location_area struct {
+		Name string
+	}
+}
+
+type pokemonStruct struct {
+	Name     string
+	Location []string
+}
+
+func parseProgramArguments(pokemon *string) {
+	flag.StringVar(pokemon, "pokemon", "1", "desired pokemon's name or number")
```

Comment:
This means that, by default, data about pokemon at index 1 will be returned. Wouldn't it be better to set the default here to an empty string, check is `pokemon` is an empty string after `parseProgramArguments` and, if it is, log an error?